src/html/partials/story/intro.hbs

```diff
@@ -28,11 +29,11 @@
 
 
 <p class="step unpad-bottom-3">
-	The authors analyzed the sonic features&#8202;—&#8202;—properties like beat, “acousticness,” and “valence,” or mood&#8202;—&#8202;for every song that has appeared on the charts since its debut in 1955. The features are calculated directly from the raw audio of the songs.
+	The authors analyzed the sonic features&#8202;—&#8202;properties like beat, “acousticness,” and “valence,” or mood&#8202;—&#8202;for every song that has appeared on the charts since its debut in 1955. The features are calculated directly from the raw audio of the songs.
 </p>
 
 <p class="step unpad-bottom-2" data-song="Believe" data-hide_avg="1">
-	Let's go over some of those features. As an example, I'll be using “Believe” by Cher, which held the #1 spot on the Billboard Hot 100 for four weeks in 1999.
```

Comment:
I think the "some of" language was because these visualizations are a subset of the features used in the paper (I dropped key, mode, time signature, and possibly 1 or 2 others). But it's probably clearer to elide that detail until the appendix.